spec/helpers/produce_items_helper_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ProduceItemsHelper, type: :helper do
+  shared_context 'when viewing the produce items index' do
+    let!(:produce_item_multiple) { create(:produce_item) }
+    let!(:produce_item_one) { create(:produce_item) }
+    let(:produce_item_empty) { create(:produce_item) }
+
+    before do
+      create(:item_season, produce_item: produce_item_multiple, month_index: 1)
+      create(:item_season, produce_item: produce_item_multiple, month_index: 2)
+      create(:item_season, produce_item: produce_item_one, month_index: 3)
+    end
+  end
+
+  describe '#formatted_month_name' do
+    include_context 'when viewing the produce items index'
+
+    describe 'when produce item has multiple seasons' do
+      it 'formats months using / as a separator' do
+        expect(helper.formatted_month_list(produce_item_multiple)).to eq('janeiro / fevereiro')
+      end
+    end
+
+    describe 'when produce item has one season' do
+      it 'returns the month name' do
+        expect(helper.formatted_month_list(produce_item_one)).to eq('março')
+      end
+    end
+
+    describe 'when produce item has no seasons' do
+      it 'returns an empty string' do
+        expect(helper.formatted_month_list(produce_item_empty)).to eq('')
+      end
+    end
+  end
```

Comment:
**suggestion:** we could leverage the block scopes to simplify the tests and make the tests slightly faster by doing less DB writes (writing this considering we tweaked Rubocop already):

```rb
  describe '#formatted_month_name' do
    # No need for ! here, since the variable will be used in the before blocks
    # We only need ! when we want the thing executed BEFORE its first usage
    let(:produce_item) { create(:produce_item) }

    describe 'when produce item has multiple seasons' do
      before do
        create(:item_season, produce_item: produce_item, month_index: 1)
        create(:item_season, produce_item: produce_item, month_index: 2)
      end
      
      it 'formats months using / as a separator' do
        expect(helper.formatted_month_list(produce_item)).to eq('janeiro / fevereiro')
      end
    end

    describe 'when produce item has one season' do
      before do
        create(:item_season, produce_item: produce_item, month_index: 1)
      end
      
      it 'returns the month name' do
        expect(helper.formatted_month_list(produce_item)).to eq('março')
      end
    end

    describe 'when produce item has no seasons' do
      it 'returns an empty string' do
        expect(helper.formatted_month_list(produce_item)).to eq('')
      end
    end
  end
```